orangecontrib/imageanalytics/image_embedder.py

```diff
@@ -193,11 +196,21 @@ def _send_to_server(self, file_paths, image_processed_callback):
         )
 
     def _load_image_or_none(self, file_path):
-        try:
-            image = open_image(file_path)
-        except IOError:
-            log.warning("Image skipped (invalid file path)", exc_info=True)
-            return None
+	parsed_url = urlparse(file_path)
+        if is_valid_url(file_path) and (parsed_url.scheme == 'http' or parsed_url.scheme == 'https'):
```

Comment:
Perhaps we can consider using [validators.url](http://validators.readthedocs.io/en/latest/#module-validators.url) to validate the URL?